doc/source/whatsnew/v1.6.0.rst

```diff
@@ -131,7 +131,7 @@ Timezones
 
 Numeric
 ^^^^^^^
--
+- Bug in cannot apply array ufunc operation when inputs are mixed with type DataFrame and type Series (:issue:`39853`)
```

Comment:
Could you reword? Not sure what this means

Reply:
Have update